### ISSUE TITLE
[BUG] agent: MAP-4.5.1_ETH fails with "No Channel Preference Report message found"

### ISSUE THREAD

**Issue by rmelotte** [prplfoundation/prplMesh#961]
Job: https://gitlab.com/prpl-foundation/prplMesh/-/jobs/468547961

logs: https://gitlab.com/prpl-foundation/prplMesh/-/jobs/468547961/artifacts/browse/logs/FAIL/MAP-4.5.1_ETH/

**Comment by tomereli** [comment]
Analysis - looks like a bug in reading ACS report, I think it something that happens from time to time but its not conclusive. In any case its not a prplMesh issue:
```
DEBUG 00:56:04:744 <1988042032> son_slave_thread.cpp[4422] --> Received CHANNEL_PREFERENCE_QUERY_MESSAGE, mid=6387
TRACE 00:56:04:746 <1983925496> ap_wlan_hal_dwpal.cpp[1441] --> read_acs_report for interface: wlan2
DEBUG 00:56:04:747 <1983925496> ap_wlan_hal_dwpal.cpp[1455] --> GET_ACS_REPORT replylen=0
DEBUG 00:56:04:748 <1983925496> ap_wlan_hal_dwpal.cpp[1456] --> GET_ACS_REPORT reply=

ERROR 00:56:04:749 <1983925496> ap_wlan_hal_dwpal.cpp[1486] --> DWPAL parse error ==> Abort
```
We should see if this happens every night and debug it further.

**Comment by tomereli** [comment]
Ran manually and passes. But fails each time in CI setup -> needs to debug in CI setup

**Comment by tomereli** [comment]
Analysis:
```
DEBUG 00:51:41:967 <1990470904> ap_wlan_hal_dwpal.cpp[1455] --> GET_ACS_REPORT replylen=0
DEBUG 00:51:41:968 <1990470904> ap_wlan_hal_dwpal.cpp[1456] --> GET_ACS_REPORT reply=

ERROR 00:51:41:968 <1990470904> ap_wlan_hal_dwpal.cpp[1486] --> DWPAL parse error ==> Abort
ERROR 00:51:41:969 <1990470904> ap_manager_thread.cpp[730] --> Failed to read acs report
```

Which means that `hostapd_cli -iwlan0 acs_report` would also fail - @rmelotte can you add this as well to the CI so we can verify this happens whenever this issue reproduces?

IMO this is a driver / hostapd error which should be resolved once #889 is done.

Edit - scratch that, we think the reason for this issue is that the ACS report is old and gets deleted - see https://github.com/prplfoundation/prplMesh/pull/941#issuecomment-601057255

**Comment by rmelotte** [comment]
Fixed by switching each radio channel back to 'auto' on the device (after this change `hostapd_cli -iwlan0 acs_report` should no longer be empty):
https://gitlab.com/prpl-foundation/prplMesh/-/jobs/477466736
